### PR TITLE
This PR adds script to check nodes status

### PR DESCRIPTION
**Pull request by Shubham-tiwari123**
Adds python script to check the besu node status

### CODE REVIEW

File: monitor besu testnet/test_node.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import subprocess
```
Comment (koshikraj) [inline monitor besu testnet/test_node.py]: Rename the folder name to `monitor`. Don't use spaces while naming directories in the future. Instead, you can use underscores.

File: monitor besu testnet/test_node.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import os
+import requests
+import time
+
+# stores besu-sample-networks folder location.
+directoryLocation = "../../../Documents/besu-sample-networks/"
```
Comment (koshikraj) [inline monitor besu testnet/test_node.py]: Can we accept all the values such as `directoryLocation`, `url`, `waitingTime`, `sleepTime` as python command line arguments? It's easier to run the file as service with command like args.

File: monitor besu testnet/test_node.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+while (True):
+	print("Getting api status..... \n")
+	try:
+		response = requests.get(url)
+		responseData = response.json()
+		print(responseData)
+		resultArray = responseData["data"]["result"]
+		print(len(resultArray))
+		if len(resultArray)!=6:
```
Comment (koshikraj) [inline monitor besu testnet/test_node.py]: You can also make number of nodes (6) as a variable and take it as arg.

File: monitor besu testnet/test_node.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+			# remove first parameter sudo if docker does not need sudo to perform
+			subprocess.call(["sudo","./stop.sh"])
+			time.sleep(sleepTime)
+			if resumeCount<2:
+				print("\nResuming.......")
+				# remove first parameter sudo if docker does not need sudo to perform
+				subprocess.call(["sudo","./resume.sh"])
+				resumeCount = resumeCount+1
+				time.sleep(waitingTime)
+	except:
```
Comment (koshikraj) [inline monitor besu testnet/test_node.py]: Catch the type of error and then decide to pass or remove the containers based not the error type.

File: monitor besu testnet/test_node.py (hunk 2 above)
Comment (Shubham-tiwari123) [reply monitor besu testnet/test_node.py]: I guess we can. let me try

File: monitor besu testnet/test_node.py (hunk 1 above)
Comment (Shubham-tiwari123) [reply monitor besu testnet/test_node.py]: ok

**Comment by Shubham-tiwari123** [comment]
Command to run the file : python3 test_node.py /root/besu/besu-sample-networks/ 0 180 10 6
First arg: location
Second arg: resume_count
Third arg: waiting_time
Fourth arg: sleep_time
Fifth arg: total_nodes

**Comment by koshikraj** [comment]
Nicely written script with error handling :+1:. LGTM.

**Pull request merged**